sahi/utils/mot.py

```diff
@@ -82,8 +82,9 @@ def __init__(self, bbox: List[int], track_id: Optional[int] = -1, score: Optiona
 
 
 class MotFrame:
-    def __init__(self):
+    def __init__(self, file_name: Optional[str] = ""):
```

Comment:
it is better to keep default value as `None` instead of `""`: https://google.github.io/styleguide/pyguide.html#2124-decision